htdocs/libraries/icms/Autoloader.php

```diff
@@ -8,6 +8,9 @@
  * @since	1.3
  * @author	Marc-André Lanciault (aka marcan) <mal@inboxintl.com>
  */
+
+include_once ICMS_ROOT_PATH . "/vendor/autoload.php"
+
```

Comment:
There is a fatal error at this point, as pointed out by Scrutinizer. There is a missing ; at the end of the line